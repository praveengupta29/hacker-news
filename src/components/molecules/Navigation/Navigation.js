import React from 'react';

import './Navigation.scss';
import NavItem from '../../atoms/NavItem';

const Navigation = () => {
  return (
    <div className="navigation-bar flex align-center">
      <NavItem>new</NavItem>
    </div>
  );
};

export default Navigation;
