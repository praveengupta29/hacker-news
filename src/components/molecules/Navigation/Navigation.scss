.navigation-bar {
  margin-left: 10px;
}
