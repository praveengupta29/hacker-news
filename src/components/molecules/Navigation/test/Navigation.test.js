import React from 'react';
import { shallow } from 'enzyme';
import Navigation from '../Navigation';

describe('Navigation component', () => {
  it('should render correctly and match snapshot', () => {
    const component = shallow(<Navigation />);
    expect(component).toMatchSnapshot();
  });
});
