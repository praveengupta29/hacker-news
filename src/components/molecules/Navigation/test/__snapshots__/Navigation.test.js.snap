// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation component should render correctly and match snapshot 1`] = `
<div
  className="navigation-bar flex align-center"
>
  <NavItem>
    top
  </NavItem>
</div>
`;
